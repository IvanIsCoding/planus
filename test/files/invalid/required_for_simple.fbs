table Foo {
  x: uint8 (required);
}