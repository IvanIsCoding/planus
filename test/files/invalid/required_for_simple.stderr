[0m[1m[38;5;9merror[0m[1m: Unsupported required attribute[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/required_for_simple.fbs:2:13
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: uint8 ([0m[31mrequired[0m);
  [0m[34mâ”‚[0m      [0m[34m-----[0m  [0m[31m^^^^^^^^[0m [0m[31mrequired attribute was here[0m
  [0m[34mâ”‚[0m      [0m[34mâ”‚[0m       
  [0m[34mâ”‚[0m      [0m[34monly non-scalar types support the 'required' attribute[0m

Error: Bailing because of errors
